### PR TITLE
This makes the headline of the post list page customizable.

### LINKED ISSUE #166: Make title on default template customizable
Looking into https://github.com/Kikobeats/uno-zen/blob/570d08c8dd1bedba0912288c1dccb6f547713cb7/index.hbs#L3 indicates that the headline of the index is fixed.
Adjusting of the Profile title is supported like documented in https://github.com/Kikobeats/uno-zen/blob/master/DOCUMENTATION.md#profile-title. Maybe also the headline of the index can be made configurable this way.

### PR DESCRIPTION
**Pull request by waja**
(Closes #166)

### CODE REVIEW

File: assets/js/src/__init.coffee [hunk 1]
```diff
@@ -43,5 +43,6 @@ Uno.app.dataset.device = Uno.device()
 # window global properties
 $('#profile-title').text window.profile_title if window.profile_title
 $('#profile-resume').text window.profile_resume if window.profile_resume
+$('#post-list-headline').text window.post_list_headline if window.post_list_headline
```
Comment (Kikobeats) [inline assets/js/src/__init.coffee]: Do you think that `posts_headline` could be a better variable name?

Same for id selector... what do you think?

Comment (waja) [inline assets/js/src/__init.coffee]: I've no strong bindings to the name. I'll rework the branch.

**Comment by waja** [comment]
This PR is updated to make use of "posts-headline" and "posts_headline" now.

**Comment by Kikobeats** [comment]
Looks better, I add a little comment to be ready to merge :-)

**Comment by waja** [comment]
Okay ... it's updated.

**Pull request merged**